package.json

```diff
@@ -1,55 +1,48 @@
 {
-  "name": "usage-analytics-app",
-  "version": "1.0.0",
-  "private": true,
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "lint:style": "d2-style js apply --all",
-    "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
-    "localize": "yarn extract-pot && d2-i18n-generate -n usage-analytics-app -p ./i18n/ -o ./src/locales/",
-    "prestart": "yarn localize",
-    "prebuild": "yarn localize",
-    "pretest": "yarn localize"
-  },
-  "dependencies": {
-    "@dhis2/app-runtime": "1.3.0",
-    "@dhis2/ui-core": "3.4.0",
-    "@dhis2/ui-widgets": "1.0.8",
-    "chart.js": "^2.7.3",
-    "core-js": "^3.0.1",
-    "lodash.debounce": "^4.0.8",
-    "react": "^16.8.6",
-    "react-app-polyfill": "^1.0.1",
-    "react-chartjs-2": "^2.7.4",
-    "react-dom": "^16.8.6",
-    "react-redux": "^7.0.3",
-    "react-scripts": "^3.0.0",
-    "redux": "^4.0.1",
-    "redux-thunk": "^2.3.0",
-    "typeface-roboto": "^0.0.54"
-  },
-  "devDependencies": {
-    "@dhis2/cli-style": "4.1.0",
-    "@dhis2/d2-i18n": "^1.0.4",
-    "@dhis2/d2-i18n-extract": "^1.0.7",
-    "@dhis2/d2-i18n-generate": "^1.1.1",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.12.1",
-    "jest-enzyme": "^7.0.2",
-    "prop-types": "^15.7.2",
-    "redux-logger": "^3.0.6",
-    "redux-mock-store": "^1.5.3"
-  },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/**/*.d.ts",
-      "!src/serviceWorker.js",
-      "!src/index.js",
-      "!src/store.js",
-      "!src/__mockData__/**/*.js"
-    ]
-  }
+    "name": "usage-analytics-app",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "start": "d2-app-scripts start",
+        "build": "d2-app-scripts build",
+        "test": "d2-app-scripts test",
+        "lint:style": "d2-style js apply --all"
```

Comment:
This is a sidenote, but I see we're running this on CI as well. I think it'd be better to use `d2-style js check --all` in that case. To avoid fixing files that are in violation of our linting rules on CI without committing them back to the repo.

Reply:
Agreed, see comment below!  Let's do this in a separate PR though.